content/docs/command-reference/params/index.md

```diff
@@ -0,0 +1,154 @@
+# params
+
+Contains a command to show changes in parameters:
+[diff](/doc/command-reference/params/diff).
+
+## Synopsis
+
+```usage
+usage: dvc params [-h] [-q | -v] {diff} ...
+
+positional arguments:
+  COMMAND
+    diff         Show changes in params between commits in the
+                 DVC repository, or between a commit and the workspace.
+```
+
+## Description
+
+In order to track parameters and hyperparameters associated to machine learning
+experiments in <abbr>DVC projects</abbr>, DVC provides a special type of
+<abbr>dependencies</abbr>: _parameters_. Parameter values are specific to a
+version of the project, and defined (using `dvc run`) with simple names like
+`epochs`, `learning-rate`, `batch_size`, etc.
+
+In contrast to a regular text file dependency, a parameter dependency consists
+of a parameters _file_ (the file dependency itself) and a parameter _name_ (to
+find inside the text file). Multiple parameter dependencies can be specified
+from one or more parameters files.
+
+The default parameters file name is `params.yaml`. Parameters should be
+organized as a tree hierarchy in it, as DVC will locate param names by their  
+tree path. Supported file formats for params files are: YAML and JSON.
+
+Supported parameter _value_ types are: string, integer, float, and arrays. DVC
+itself does not ascribe any specific meaning for these values. They are
+user-defined, and serve as a way to generalize and parametrize an machine
+learning algorithms or data processing code.
```

Comment:
The only question I still have is about this paragraph. What do we mean supported? Isn't any valid YAML or JSON value automatically supported? Supported for what, for `params diff`? If so maybe we should only mention this in that other doc (I did copy part of this p to there already). @dmpetrov